### DISCUSSION TITLE
第6节课笔记

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#471] [笔记]
笔记贴在这里

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture6.md

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_6.md

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/blob/main/Lecture6_OpenCompass

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/hr7g3wezn2xff572?singleDoc#

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture6-OpenCompass-a785b001db164e6fb31b0c0546048f10?pvs=4

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/62ec786a-9992-4f18-a8d6-6919451bc2b3
